portfolio/src/main/webapp/job-experience/job-experience.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../default.css">
+    <link rel="stylesheet" href="../buttons.css">
+    <link rel="stylesheet" href="job-experience.css">
+  </head>
+  <body>
+    <div id="description-placeholder">
+      <div class="description">
+        Hover over the image to read the description!
+      </div>
+    </div>
+    <div>
+      <div class="img-bubble" id="oij-bubble">
+        <img src="images/oij-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I'm part of jury comitee of Junior Olympiad in Informatics.
+          I prepare tasks and create tests set for competitors.
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="img-bubble" id="samsung-bubble">
+        <img src="images/samsung-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          During summer 2019 I was working on open-source operating
+          system -- Tizen.
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="img-bubble" id="google-bubble">
+        <img src="images/google-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I'm Google STEP intern this year! This website is my first
+          assignment : )
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="img-bubble" id="staszic-bubble">
+        <img src="images/staszic-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I really love teaching!
+          I was conducting Algorithms & Data Structures classes in
```

Comment:
nit: while a free-standing ampersand is unambiguous and thus allowed in HTML5, some people say that it's a good practice is to escape it (`&amp;`) anyway

Reply:
Done.